src/components/ReviewsComponent/SingleReviewComponent.jsx

```diff
@@ -12,16 +12,6 @@ const SingleReviewComponent = ({
   description,
 }) => {
   const language = useSelector(state=> state.languageReducer);
-  const reviewBy = () => {
-    switch (language) {
-      case 'en':
-        return en.reviewBy
-      case 'es':
-        return es.reviewBy
-      default:
-        return en.reviewBy
-    }
-  }
   const stars = (rating) => {
```

Comment:
This star generator, is not bad, but if there is time for experimenting, consider a proposal enhancement to adapt:
[star generator](https://github.com/codegeous/react-component-depot/blob/master/src/components/Rate/index.js)

Reply:
Please, consider this enhancement in a future PR, if it's possible.